sdkproject/Assets/Mapbox/Examples/1_DataExplorer/Scripts/TextMeshOutline.cs

```diff
@@ -0,0 +1,91 @@
+﻿namespace Mapbox.Examples
```

Comment:
Why is this  file included here?

Reply:
This renders an outline on the text in the Data Explorer scene